src/binding-language.js

```diff
@@ -79,19 +79,17 @@ export class TemplatingBindingLanguage extends BindingLanguage {
   }
 
   parseContent(resources, attrName, attrValue){
-    var parts = attrValue.split(this.interpolationRegex), i, ii;
-    if (parts.length <= 1) { //no expression found
+    var parts = [];
+    parse(attrValue, (type, start, end, extra) => {
+      var value = attrValue.substring(start, end) + (extra || '');
+      var expr = type === 'expr' ? this.parser.parse(value) : null;
+      parts.push({type, value, expr});
+    });
+
+    if (exprs.length == 0) { //no expression found
```

Comment:
I'm currently not able to run the code (asside from executing the tests), but this should in theory at least resolve the problem. It's doe to rewrite I did between the first and second commit. If you're able to run it, and it works, then great. Otherwise I'll have to look at it tomorrow.